contrib/platform/src/com/sun/jna/platform/win32/WinUser.java

```diff
@@ -492,6 +518,27 @@ public SIZE(int w, int h) {
     public interface HOOKPROC extends StdCallCallback { }
 
     /**
+     * Defines the message parameters passed to a WH_CALLWNDPROC hook procedure, CallWndProc.
+     */
+    public class CWPSTRUCT extends Structure {
+
```

Comment:
While the structure is generally passed in from outside, I'd add a no-arg constructor for completeness.